database/postgres/dal/user.py

```diff
@@ -28,4 +29,24 @@ def get_user_by_id(self, user_id: int) -> User:
 
     def update_user_password(self, user: User, new_password):
         user.password_hash = new_password
-        self.session.commit()
\ No newline at end of file
+        self.session.commit()
+
+    def get_user_id_be_email(self, email: str) -> User:
```

Comment:
get user ID but return the whole user.

Either rename or return just an ID (select only ID from database, don't do that in python)